frontend/src/composables/useFetch.ts

```diff
@@ -61,29 +79,27 @@ export const useFetchTo = (
       data.value = result.data;
     } catch (ex) {
       error.value = ex;
-      if (
-        error.value.code === "ERR_BAD_RESPONSE" ||
-        error.value.code === "ERR_NETWORK"
-      ) {
-        notificationBus.emit({
-          fieldId: "internal.server.error",
-          errorMsg: "",
-        });
-      }
-      else if (error.value.code === "ERR_BAD_REQUEST") {
-        notificationBus.emit({
-          fieldId: "bad.request.error",
-          errorMsg: "",
-        });
+      if (config.skipDefaultErrorHandling) {
+        return;
       }
+      thisObj.handleErrorDefault();
     } finally {
       loading.value = false;
     }
   };
 
   !config.skip && fetch();
 
-  return { response, error, data, loading, fetch };
+  const thisObj = {
```

Comment:
@fterra-encora could you please give a meaningful name to "thisObj"?

Reply:
done